### PR TITLE
Update /react-native-picker/LICENSE spelling.

### PR DESCRIPTION
**Pull request by QuietEvolver**
# Summary

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

## Checklist

- [ ] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (EvanBacon) [inline LICENSE:1]: Be sure to delete the other file too

Comment (QuietEvolver) [reply LICENSE]: Done. Thanks for catching that.

**Comment by QuietEvolver** [comment]
- My pleasure.

**Pull request merged**